### PR TITLE
i18n-text-input

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

-----

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18731/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18732/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/19033/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/19158/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/19204/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21312/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by jeremie-lucca** [comment]
- [ ] Check if we cover disabled state / disabled UI
- [ ] Add "Traduire le texte" tooltip on translation button
- [ ] Add ellipsis on Textfield value + tooltip (to discuss)
- [ ] Error / Warning / Success states to be discussed

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21325/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21417/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21562/ui-diff)

File: packages/ng/forms/multilanguage-input/multilanguage-input.translate.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		clear: 'Empty this field',
+		toggleMultilanguage: 'Show translations',
+		translateTo: 'Translate to {{lang}}',
+	},
+	fr: {
+		clear: 'Vider ce champ',
+		toggleMultilanguage: 'Afficher les traductions',
+		translateTo: 'Traduire en {{lang}}',
+	},
+};
```
Comment (jbosi) [inline packages/ng/forms/multilanguage-input/multilanguage-input.translate.ts:25]: On ne passe pas par lokalise pour les traductions de ce composant ?

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21568/ui-diff)

File: packages/ng/forms/multilanguage-input/multilanguage-input.translate.ts (hunk 1 above)
Comment (jeremie-lucca) [reply packages/ng/forms/multilanguage-input/multilanguage-input.translate.ts:25]: Si tu as raison, le composant a été fait avant le passage à Lokalise.
Par contre c'est Guillaume qui a la marche à suivre et il est en vacances. 
Je vais virer la trad et l'ajouter à cette PR pour son retour : https://github.com/LuccaSA/lucca-front/issues/2990

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21666/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21668/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Review by jeremie-lucca — APPROVED** [review]

File: packages/ng/forms/multilanguage-input/multilanguage-input.component.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@use '@lucca-front/scss/src/commons/utils/a11y';
+
+.popover-contentOptional {
+
+	padding: var(--pr-t-spacings-100) !important;
+	max-height: 20rem;
```
Comment (fbasmaison-lucca) [inline packages/ng/forms/multilanguage-input/multilanguage-input.component.scss:6]: Can we imagine `max-height` being a variable?

File: packages/ng/forms/multilanguage-input/multilanguage-input.component.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@use '@lucca-front/scss/src/commons/utils/a11y';
+
+.popover-contentOptional {
+
```
Comment (fbasmaison-lucca) [inline packages/ng/forms/multilanguage-input/multilanguage-input.component.scss:4]: Though harmless, I’m not sure this extra line follows any styleguide. :)

```suggestion
```

File: docs/dialog.md [hunk 4]
```diff
@@ -177,14 +177,14 @@ la dialog box a accès à tout ce que le composant expose au template, fonctions
 
 ## Utilisation des formulaires
 
-Dans le cas où vous souhaitez utiliser un formulaire au sein d’une dialog box, il vous suffit de placer la balise `form` avec une classe `dialog-formOptionnal` autour du contenu de votre dialog, que celle-ci soit template-driven ou non:
+Dans le cas où vous souhaitez utiliser un formulaire au sein d’une dialog box, il vous suffit de placer la balise `form` avec une classe `dialog-formOptional` autour du contenu de votre dialog, que celle-ci soit template-driven ou non:
```
Comment (fbasmaison-lucca) [inline docs/dialog.md:180]:
```suggestion
Dans le cas où vous souhaitez utiliser un formulaire au sein d’une [dialog box](https://prisme.lucca.io/94310e217/p/841b0b-dialogs), il vous suffit de placer la balise `form` avec une classe `dialog-formOptional` autour du contenu de votre dialog, que celle-ci soit template-driven ou non :
```

- Link to the resource we’re mentioning.
- Non-breakable space before `:`.

File: packages/ng/forms/multilanguage-input/multilanguage-input.component.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+				<div class="popover-contentOptional" [style.width.px]="inputElement.clientWidth + 16">
+					<div class="u-displayFlex u-gapXS u-flexDirectionColumn">
+						@for (row of panelInputs(); track row.cultureCode) {
+						<lu-form-field [label]="intl.translateTo | intlParams: {lang: getLocaleDisplayName(row.cultureCode)}" hiddenLabel>
+							<lu-text-input
+								[suffix]="{content: row.cultureCode, ariaLabel: ''}"
+								[(ngModel)]="row.value"
+								(ngModelChange)="valueChange()"
+								[disabled]="disabledInternal()"
+							></lu-text-input>
```
Comment (fbasmaison-lucca) [inline packages/ng/forms/multilanguage-input/multilanguage-input.component.html:34]: Use a [consistent ordering of attributes](https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/docs/rules/attributes-order.md)?

```suggestion
		<input
			#inputElement
			class="textField-input-value"
			type="text"
			luInput
			[disabled]="disabledInternal()"
			[placeholder]="placeholder()"
			[(ngModel)]="invariant().value"
			(ngModelChange)="valueChange()"
			(blur)="onTouched?.()"
		/>
		<div class="textField-input-affix">
			<button
				class="button mod-onlyIcon mod-text textField-input-affix-toggle"
				type="button"
				[customPositions]="popoverPositions"
				[luPopover2]="popoverMultilanguage"
				[luTooltip]="intl.toggleMultilanguage"
			>
				<span class="lucca-icon icon-signTranslation" aria-hidden="true"></span>
			</button>
			<ng-template #popoverMultilanguage>
				<div class="popover-contentOptional" [style.width.px]="inputElement.clientWidth + 16">
					<div class="u-displayFlex u-gapXS u-flexDirectionColumn">
						@for (row of panelInputs(); track row.cultureCode) {
						<lu-form-field [label]="intl.translateTo | intlParams: {lang: getLocaleDisplayName(row.cultureCode)}" hiddenLabel>
							<lu-text-input
								[disabled]="disabledInternal()"
								[suffix]="{content: row.cultureCode, ariaLabel: ''}"
								[(ngModel)]="row.value"
								(ngModelChange)="valueChange()"
							></lu-text-input>
```

File: packages/ng/forms/multilanguage-input/multilanguage-input.component.scss (hunk 2 above)
Comment (jeremie-lucca) [reply packages/ng/forms/multilanguage-input/multilanguage-input.component.scss:6]: If you see use cases, yep!

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2890/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21688/ui-diff)

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/21689/ui-diff)

**Pull request merged**